### ISSUE TITLE
Default testplatform config should be part of vsix

### ISSUE THREAD

**Issue by codito** [Microsoft/vstest#124]
Test explorer uses below config file to read features to enable for TPv2. The default vsix should include `testplatform.config`.

```
<?xml version="1.0" encoding="utf-8"?>
<configuration>
  <appSettings>
    <add key="feature.net40" value="false" />
  </appSettings>
</configuration>
```

**Comment by sharwell** [comment]
Was this fixed in #713?

**Comment by codito** [comment]
Yes, it should be part of upcoming VS releases. We can close this issue.
